src/shared/router/routes/index.ts

```diff
@@ -304,6 +304,12 @@ const routes = [
         component: AmendEntity(mpamCampaign),
         title: 'Amend Campaign'
     },
+    {
+        path: '/manage-trof-campaigns/:itemUUID/amend',
```

Comment:
Is there a test for this path?

Reply:
The router isn't tested at all at the moment - for any of the paths. I'll create a test which exercises the router for the TROF campaigns paths.

Reply:
I've added these now